### ISSUE TITLE
Missing ICD-10-CM diagnosis codes

### ISSUE THREAD

**Issue by gowthamrao** [OHDSI/Vocabulary-v5.0#124]
Hello - these ICD-10-CM codes does not seem to have a concept-id assigned to them. Maybe they were missed? Some of them are pretty important ones like 'Pure Hypercholesterolemia'. I tried to order them based on priority and am presenting about 250 codes for considerations.

ps: there is no '.' in the attached ICD10 codes, but the original ICD10 codes will have a '.'

[ICD-10-CM code missing.xlsx](https://github.com/OHDSI/Vocabulary-v5.0/files/1025835/ICD-10-CM.code.missing.xlsx)

**Comment by gowthamrao** [comment]
Yes - thank you

**Comment by ericaVoss** [comment]
Just making a note that this is still not fixed in 20170503 (as you already stated above) and so I can store this test query.

```{SQL}
SELECT *
FROM CONCEPT
WHERE VOCABULARY_ID = 'ICD10CM'
AND REPLACE(CONCEPT_CODE,'.','') IN (
'E7800','Z98890','R9720','Z7984','R7303','R3129','H401131','M50322','K8590','N83201','H401132','F3289','N83202','F429','O34211','O34219','G5603','N83209','F422','N9410','M50323','H353131','M50222','E7801','R3121','I160','K0889','H401133','M50320','M50321','E113293','H353211','M50223','H353221','M50122','N5089','K8689','N610','H353132','M50220','N611','K5904','M50123','M50221','H353231','M25541','N83291','N50819','K5903','E113513','F5081','F3481','N50812','H401122','H401111','K581','M25542','H401112','K8510','N83292','M26609','H401121','Z3046','R39198','Z516','N50811','I161','M21611','M21612','Z30017','M50121','R8271','O134','E113593','E113313','H401123','K582','N5082','H401113','F5089','H353112','M50120','H353130','K8520','E113511','L03213','K8500','F3281','F428','H353122','K9049','N8302','O24415','H401130','E113512','N8301','M50022','H348310','O0090','H348320','H401134','N83299','Z98891','M26629','E113393','N9412','H353111','Z299','E113299','H353121','H353212','H348110','E113311','M25549','E113312','X500XXA','E113213','H348120','E113413','F640','N8311','O0010','E113291','K8591','H353222','N8312','E113592','K8580','N9419','E113591','J9859','D4959','K588','I169','E113292','F3489','K52832','M50023','K8681','X501XXA','M26621','H353110','H348312','M26622','T82855A','Z5331','Z3044','X509XXA','M26623','N8300','K5229','R9721','M50021','E103593','M26601','O1414','H353232','R402411','O7020','E113412','T83511A','H348322','R9341','M21621','M26603','M21622','M26602','G5793','R8279','L7634','Z051','E113411','N9411','K52831','Z30015','L7632','E113391','E103293','H353133','M50020','E113211','O4442','O24425','H90A22','E103513','K523','E113212','S0232XA','M21619','H348122','O1404','E113392','H353230','O1494','H353210','H401192','H348112','Q6621','G5623','K8511','H353134','Z8342','H90A32','R402412','E113599','Z293','H353120','H90A21','H353220','K9041','H353223','H401191','H401190','N8310','H353290','C49A0','I69319','Q6622','H353190','E113553','H401110','H90A31','O4443','R9349','M50822','S0231XA','H353213','W268XXA','V475XXA','H353124','Z191','O114','N9060','E103511','N4231','F424','R93421','T85848A','S0300XA','R402431','M26633','E113493','T85898A','Q826','H401120','K52839'
)
```

**Comment by ericaVoss** [comment]
In 20170920 this issue is solved.  All 246 are in the Vocab now. 

Could we please close the issue @gowthamrao ?

**Comment by ericaVoss** [comment]
I believe this is still solved still in 20171201.  @gowthamrao or @dimshitc  can you please close this issue?
